Kickstarter-iOS/AppDelegate.swift

```diff
@@ -384,9 +384,8 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
   // MARK: - Qualtrics Configuration
 
   private func configureQualtrics(with config: QualtricsConfigData) {
-    let properties = Properties()
```

Comment:
This clearly wasn't correct - I'm not sure how we got here 🤔

Reply:
Ha it wasn't doing anything before 🤦‍♀. Do you think it matters that we set the properties on the shared instance _before_ initializing it? 🤔

Reply:
This way seems to work.